src/main/java/frc/sensors/Limelight.java

```diff
@@ -1,9 +1,29 @@
 package frc.sensors;
 
+import java.lang.Math;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
 public class Limelight {
 	
+	private NetworkTable table;
+	NetworkTableEntry ty;
+	
+	public Limelight() {
+		table = NetworkTableInstance.getDefault().getTable("limelight");
+		ty = table.getEntry("ty");
+	}
+
 	public double getDistanceFromTarget() {
-		return 0;
+		
+		final double a1 = 15;

```

Comment:
It may help if you make these variable names a little most clear on what they represent